conans/test/functional/toolchains/google/test_bazel.py

```diff
@@ -34,81 +34,59 @@ def base_profile():
         """)
 
 
-@pytest.fixture(scope="module")
-def client_exe(bazelrc):
+@pytest.mark.parametrize("build_type", ["Debug", "Release", "RelWithDebInfo", "MinSizeRel"])
+@pytest.mark.tool_bazel
+def test_basic_exe(bazelrc, build_type, base_profile):
     client = TestClient(path_with_spaces=False)
     # client.run("new bazel_exe -d name=myapp -d version=1.0")
     client.run("new myapp/1.0 -m bazel_exe")
     # The build:<config> define several configurations that can be activated by passing
     # the bazel config with tools.google.bazel:configs
     client.save({"mybazelrc": bazelrc})
-    return client
-
-
-@pytest.fixture(scope="module")
-def client_lib(bazelrc):
```

Comment:
Removed because it's enough tested by the next test `test_transitive_libs_consuming`.